.fixtures.yml

```diff
@@ -1,8 +1,11 @@
 fixtures:
   repositories:
-    stdlib: 
-      repo: 'git://github.com/puppetlabs/puppetlabs-stdlib.git'
-    concat:
-      repo: 'https://github.com/puppetlabs/puppetlabs-concat.git'
+    apt: 'https://github.com/puppetlabs/puppetlabs-apt.git'
+    concat: 'https://github.com/puppetlabs/puppetlabs-concat.git'
+    stdlib: 'https://github.com/puppetlabs/puppetlabs-stdlib.git'
+    translate: 'https://github.com/puppetlabs/puppetlabs-translate.git'
```

Comment:
why is translate needed?

Reply:
It's a [dependency needed by apt](https://github.com/puppetlabs/puppetlabs-apt/blob/master/metadata.json#L15-L18).  I did not added apt / yumrepo to the `metadata.json` file (only to the fixture files) in order to not pull-in new dependencies.

Let me know if you prefer the module to depend hardly on this "optional" dependencies.

Reply:
Understood, it's fine then